### ISSUE TITLE
relative path from css

### ISSUE THREAD

**Issue by xenlo** [kakawait/hugo-tranquilpeak-theme#390]
### Configuration

 - **Operating system with its version**: Fedora 31
 - **Browser with its version**: N/A
 - **Hugo version**: v0.58.3
 - **Tranquilpeak version**: 0.4.7-BETA
 - **Do you reproduce on https://tranquilpeak.kakawait.com demo?**: No

### Actual behaviour
The generated link to the css file is referenced from root dir (`/`):
```
<link rel="stylesheet" href="/css/style-twzjdbqhmnnacqs0pwwdzcdbt8yhv8giawvjqjmyfoqnvazl0dalmnhdkvp7.min.css" />
```

### Expected behaviour
That it behave the same way as the images, having the path generated with the `baseURL`

For example if I have `baseURL: "https://xen-lo.org/serom_3/"` in my `config.yml` it generate the following images links:
```
<img id="about-card-picture" src="https://xen-lo.org/serom_3/images/theme/avatar.png" alt="" />
[...]
<div id="cover" style="background-image:url('https://xen-lo.org/serom_3/images/theme/cover_01.jpg');"></div>
```
So for the css, I expect to have 
```
<link rel="stylesheet" href="https://xen-lo.org/serom_3/css/style-twzjdbqhmnnacqs0pwwdzcdbt8yhv8giawvjqjmyfoqnvazl0dalmnhdkvp7.min.css" />
```

### Steps to reproduce the behaviour
1. Set a hugo config file with a  `baseURL` set.
1. Generate you site with `hugo`
1. Look the the link of you stylesheet: `grep "css/style" -r public/index.html`

**Comment by xenlo** [comment]
Identified file:
- [`layouts/partials/head.html` line 48](https://github.com/kakawait/hugo-tranquilpeak-theme/blob/master/layouts/partials/head.html#L48)

I will propose a PR later.

**Comment by kakawait** [comment]
@xenlo Thank you for reporting. Will be part of next version

**Comment by xenlo** [comment]
I found a new issue of static path when you use sub directories in your `baseURL`. It's is about the `.js` file this time.

I provided a fix in PR #453
